tests/providers/amazon/aws/hooks/test_redshift_statement.py

```diff
@@ -0,0 +1,72 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import json
+import unittest
+from unittest import mock
+
+from airflow.models import Connection
+from airflow.providers.amazon.aws.hooks.redshift_statement import RedshiftStatementHook
+
+
+class TestRedshiftStatementHookConn(unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.connection = Connection(login='login', password='password', host='host', port=5439, schema="dev")
+
+        class UnitTestRedshiftStatementHook(RedshiftStatementHook):
+            conn_name_attr = "redshift_conn_id"
+            conn_type = 'redshift+redshift_connector'
+
+        self.db_hook = UnitTestRedshiftStatementHook()
+        self.db_hook.get_connection = mock.Mock()
+        self.db_hook.get_connection.return_value = self.connection
+
+    def test_get_uri(self):
+        uri_shouldbe = 'redshift+redshift_connector://login:password@host:5439/dev'
+        x = self.db_hook.get_uri()
+        assert uri_shouldbe == x
```

Comment:
```suggestion
        assert x == uri_shoudbe
```
the convention with pytest is `assert actual == expected`

nit pick here, but would also be easier to read / more conventional if you called it  `expected` or `uri_expected` instead of `uri_shouldbe`

Reply:
addressed in 01091dcace8f3c6efd57a2f7d144f7a9336ecbcd